docs/index.md

```diff
@@ -1198,53 +1312,62 @@ Options:
-| `OS_ENDPOINT_TYPE`   | `--openstack-endpoint-type` |
+Environment variables and Default value:
+
+| CLI option                    | Environment variable | Default |
+|-------------------------------|----------------------|---------|
+| `--openstack-auth-url`        | `OS_AUTH_URL`        | -       |
+| `--openstack-flavor-name`     | -                    | -       |
+| `--openstack-flavor-id`       | -                    | -       |
+| `-openstack-image-name`       | -                    | -       |
+| `--openstack-image-id`        | -                    | -       |
+| `--openstack-insecure`        | -                    | -       |
+| `--openstack-domain-name`     | `OS_DOMAIN_NAME`     | -       |
+| `--openstack-domain-id`       | `OS_DOMAIN_ID`       | -       |
+| `--openstack-username`        | `OS_USERNAME`        | -       |
+| `--openstack-password`        | `OS_PASSWORD`        | -       |
+| `--openstack-tenant-name`     | `OS_TENANT_NAME`     | -       |
+| `--openstack-tenant-id`       | `OS_TENANT_ID`       | -       |
+| `--openstack-region`          | `OS_REGION_NAME`     | -       |
+| `--openstack-endpoint-type`   | `OS_ENDPOINT_TYPE`   | -       |
+| `--openstack-net-name`        | -                    | -       |
+| `--openstack-net-id`          | -                    | -       |
+| `--openstack-sec-groups`      | -                    | -       |
+| `--openstack-floatingip-pool` | -                    | -       |
+| `--openstack-ssh-user`        | -                    | `root`  |
+| `--openstack-ssh-port`        | -                    | `22`    |
 
 #### Rackspace
 Create machines on [Rackspace cloud](http://www.rackspace.com/cloud)
 
 Options:
 
- - `--rackspace-username`: Rackspace account username
- - `--rackspace-api-key`: Rackspace API key
- - `--rackspace-region`: Rackspace region name
- - `--rackspace-endpoint-type`: Rackspace endpoint type (adminURL, internalURL or the default publicURL)
- - `--rackspace-image-id`: Rackspace image ID. Default: Ubuntu 14.10 (Utopic Unicorn) (PVHVM)
- - `--rackspace-flavor-id`: Rackspace flavor ID. Default: General Purpose 1GB
- - `--rackspace-ssh-user`: SSH user for the newly booted machine. Set to root by default
- - `--rackspace-ssh-port`: SSH port for the newly booted machine. Set to 22 by default
+ - `--rackspace-username`: **required** Rackspace account username.
+ - `--rackspace-api-key`: **required** Rackspace API key.
+ - `--rackspace-region`: **required** Rackspace region name.
+ - `--rackspace-endpoint-type`: Rackspace endpoint type (adminURL, internalURL or the default publicURL).
+ - `--rackspace-image-id`: Rackspace image ID. Default: Ubuntu 14.10 (Utopic Unicorn) (PVHVM).
+ - `--rackspace-flavor-id`: Rackspace flavor ID. Default: General Purpose 1GB.
+ - `--rackspace-ssh-user`: SSH user for the newly booted machine.
+ - `--rackspace-ssh-port`: SSH port for the newly booted machine.
+ - `--rackspace-docker-install`: Set if docker have to be installed on the machine.
```

Comment:
"Set if Docker has to be installed on the machine", although this flag might be obsolete now.